### PR TITLE
Netgear LTE binary sensors

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

* Rewords aged introduction because we now have too many sensors to enumerate
* Fixes a wrong default value
* Moves some sensors to binary sensors (the linked parent PR)
* Removes a couple of sections with no real content because I did not want to repeat it again for binary sensors

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22902

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by MartinHjelmare — COMMENTED** [review]
Add `Binary Sensor` to `ha_category` in front matter. I think that's a standard now that we add all platforms as categories. They will then show up in the component search.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/netgear_lte.markdown
Comment (klaasnicolaas) [inline source/_components/netgear_lte.markdown]: Please keep this, but delete the link and add `Binary Sensor` to the list 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Comment by amelchio** [comment]
Thanks guys!

**Pull request merged**
